Userland/Libraries/LibGfx/Painter.h

```diff
@@ -108,7 +109,12 @@ class Painter {
     };
     void fill_path(Path const&, Color, WindingRule rule = WindingRule::Nonzero);
 
-    Font const& font() const { return *state().font; }
+    Font const& font() const
+    {
+        if (!state().font)
+            const_cast<Painter*>(this)->state().font = &FontDatabase::default_font();
```

Comment:
Can we just `return FontDatabase::default_font();` ? No need for the `const_cast`

Reply:
yeah sure